wiki/Tournaments/HRS/2024/en.md

```diff
@@ -50,12 +50,31 @@ Hard Rock Showdown is run by various community members.
 
 ## Links
 
-- [**Register here**](https://forms.gle/vuTvUuE1PBZ1uFot7)
+- [Information spreadsheet](https://chillierpear.xyz/HRSMain)
```

Comment:
would rather use a direct link in order to avoid the redirect

Reply:
oke